src/commonmark/en/content/sysadmin/audit.md

```diff
@@ -4,45 +4,119 @@
+| auditid    | integer                     |                                                                                                                                                   |   |
+| audittype  | text                        | READ, CREATE, UPDATE, DELETE, SEARCH                                                                                                                  |   |
+| auditscope | text                        | METADATA, AGGREGATE, TRACKER                                                                                                                        |   |
+| klass      | text                        | Audit Entity Java class name                                                                                                                      |   |
 | attributes | jsonb                       | Json string stores attributes of the audit entity, used for searching. Example: {"valueType":"TEXT", "categoryCombo":"SWQW313FQY", "domainType":"TRACKER"} |   |
-| data       | bytea                       | Compressed Json string of the Audit Entity    
-| createdat  | timestamp without time zone |               |
-| createdby  | text                        |               | 
-| uid        | text                        |               |
-| code       | text                        |               |
+| data       | bytea                       | Compressed Json string of the Audit Entity. It is currently in byte array format and not human-readable, but a mechanism will be introduce in 2.34.1                                                                                                        |   |
+| createdat  | timestamp without time zone |                                                                                                                                                   |   |
+| createdby  | text                        |                                                                                                                                                   |   |
+| uid        | text                        |                                                                                                                                                   |   |
+| code       | text                        |                                                                                                                                                   |   |
+|            |                             |   
+
+
+
+The new Audit service makes use of two new concepts: Audit Scopes and Audit Type.
+
+## Audit Scope
+
+<!--DHIS2-SECTION-ID:audit_scope-->
+
+An Audit Scope is a logical area of the application which can be audited. Currently there are three Audit Scopes:
+
+```
+Tracker
+
+Metadata
+
+Aggregate
+```
+
+- For the Tracker Audit Scope, the audited objects are: Tracked Entity Instance, Enrollment, Event, Tracked Entity Attribute Value.
```

Comment:
The order should be: Tracked Entity, Tracked Entity Attribute Value, Enrollment, Event